crates/aptos/src/move_tool/template.rs

```diff
@@ -0,0 +1,226 @@
+use std::collections::BTreeMap;
+
+use std::fs;
+
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use std::str::FromStr;
+
+
+use anyhow::{anyhow, ensure, Context};
+use async_trait::async_trait;
+use clap::{Parser};
+use convert_case::{Case, Casing};
+use handlebars::Handlebars;
+use path_absolutize::Absolutize;
+
+use serde_json::json;
+
+use walkdir::WalkDir;
+
+use crate::common::types::{CliCommand, CliTypedResult, PromptOptions};
+use crate::move_tool::FrameworkPackageArgs;
+
+const GIT_APTOS_TEMPLATES_URL: &str = "https://github.com/mkurnikov/aptos-core.git";
```

Comment:
Is this going to be updated to point to the main aptos-core in a separate PR?

Reply:
I can do it now, just it won't be possible to test the PR before the merge then.

Reply:
UPD. Oh, yes, should be a separate PR.